### ISSUE TITLE
Nil pointer dereference when use an array of type from external package

### ISSUE THREAD

**Issue by BoyerDamien** [masaushi/accessory#51]
Step to repoduce

In `./mypackage/pkg.go`
```go
package mypackage

import "issue_accessory/pkgtwo"

//go:generate accessory -type MyPacker -receiver mp -output my_packer.go
type MyPacker struct {
	tt   []pkgtwo.MyType `accessor:"getter"`
	name string          `accessor:"getter"`
}
```

In `./pkgtwo/pkg.go`
```go
package pkgtwo

type MyType struct{}
```

Run go generate:
```bash
go generate ./...

panic: runtime error: invalid memory address or nil pointer dereference...
```
If I remove the array it works
```go
package mypackage

import "issue_accessory/pkgtwo"

//go:generate accessory -type MyPacker -receiver mp -output my_packer.go
type MyPacker struct {
	// tt   []pkgtwo.MyType `accessor:"getter"`
	name string          `accessor:"getter"`
}
```
Accessory version: `v0.1.0`

**Comment by masaushi** [comment]
@BoyerDamien 
Thank you for opening the issue! I fixed the bug and released the new version v0.1.1.
Please try it out!

**Comment by BoyerDamien** [comment]
You're welcome! Thank you for the the fix ;)
